### PR TITLE
vscode-extensions.msjsdiag.debugger-for-chrome: Init at 4.12.11

### PR DESCRIPTION
**Pull request by andyrichardson**

### CODE REVIEW

File: pkgs/misc/vscode-extensions/default.nix [hunk 1]
```diff
@@ -389,6 +389,18 @@ let
...
 
+      msjsdiag.debugger-for-chrome = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "debugger-for-chrome";
+          publisher = "msjsdiag";
+          version = "4.12.11";
+          sha256 = "sha256-9i3TgCFThnFF5ccwzS4ATj5c2Xoe/4tDFGv75jJxeQ4=";
+        };
+        meta = {
+          license = stdenv.lib.licenses.mit;
```
Comment (SuperSandro2000) [inline pkgs/misc/vscode-extensions/default.nix:400]:
```suggestion
          license = lib.licenses.mit;
```

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 109531` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>vscode-extensions.msjsdiag.debugger-for-chrome</li>
  </ul>
</details>

File: pkgs/misc/vscode-extensions/default.nix (hunk 1 above)
Comment (andyrichardson) [reply pkgs/misc/vscode-extensions/default.nix:400]: What should the commit name be for this change?

I'd love to amend the previous commit but the last time I tried to commit locally, it took 3 hours (on a fast internet connection w/ ssd) before failing

Screen Shot 2021-01-18 at 10 26 32

Comment (andyrichardson) [reply pkgs/misc/vscode-extensions/default.nix:400]: A minor suggestion to make it easier for folks to contribute in the future without commit history being a blocking issue - PRs can be squashed into a single commit pre-merge and have the commit message set.

Kapture 2021-01-18 at 10 36 04

Comment (SuperSandro2000) [reply pkgs/misc/vscode-extensions/default.nix:400]: Amending and force pushing should only take a few seconds. Also I know about this feature and certain people complain when I use it and I regularly forget to use it.

Comment (andyrichardson) [reply pkgs/misc/vscode-extensions/default.nix]: >  Also I know about this feature and certain people complain when I use it and I regularly forget to use it.

I hear you dude!

It's taking me a while to clone the repo - it's gonna create some noise but I'll have to create new PRs for now until I get around to cloning the full history.

**Pull request merged**